wallet-user/go-example/go.mod

```diff
@@ -2,21 +2,35 @@ module paymaster-example
 
```

Comment:
the readme of this package is wrong.

![image](https://github.com/user-attachments/assets/9db4f9a8-739a-439e-84c2-48394ba0b9f2)

Reply:
so is other go examples.

Reply:
fixed